test/controllers/individual/DeclarationControllerSpec.scala

```diff
@@ -89,7 +89,7 @@ class DeclarationControllerSpec extends ControllerSpecBase with MockitoSugar wit
 
       when(mockCompoundNavigator.nextPage(Matchers.eq(DeclarationPage), any(), any())).thenReturn(dummyCall)
 
-      val result = route(application, httpPOSTRequest(submitUrl, valuesValid)).value
+      val result = route(application, httpGETRequest(submitUrl)).value
```

Comment:
why its changed to GET?

Reply:
This was changed to a GET on Friday. I will change it back.